tests/validation/convergence_test.hpp

```diff
@@ -2,10 +2,11 @@
 
 #include <util/filter.hpp>
 #include <util/rangeutil.hpp>
+#include <cell.hpp>
 
 #include <json/json.hpp>
 
-#include "gtest.h"
+#include "../gtest.h"
```

Comment:
The `tests/CMakeLists.txt` file adds the directory with `gtest.h` (and the `test_common_cells.hpp` etc.) to the include path. Use of implicit include path vs. relative-to-source path in the test sources is inconsistent, but there shouldn't be any need to change code to use the relative path if it's already correctly finding `gtest.h` inter alia.